descriptors/ModuleDescriptor-template.json

```diff
@@ -20,6 +20,7 @@
           "methods": ["POST"],
           "pathPattern": "/orders/composite-orders",
           "permissionsRequired": ["orders.item.post"],
+          "permissionsDesired": ["orders.assignments.item.post"],
```

Comment:
The permission name might be confusing because it does not highlight that this is for acq units...

Reply:
renamed